root/package.json

```diff
@@ -26,9 +26,27 @@
     "jstransformer-marked": "1.0.x",
     <% if (netlify == true) { %>"roots": "3.x",<%} -%>
     <% if (yaml == true) { %>"roots-yaml": "0.0.x",<%} -%>
-    <% if (grid == true) { %>"jeet": "6.1.x",<%} -%>
-    "rupture": "0.6.x",
-    "stylus": "0.52.x"
+    <% if (grid == true) { %>
+      "jeet": "6.1.x",
+      "rupture": "0.6.x",
+      "stylus": "0.52.x",
+      "lost": "6.6.x",
+    <%} -%>
+    <% for(var i = 0; i < postcssPLugins.length; i++) { %>
+      <% if (postcssPlugins[i].checked) { %>
+        <% if (postcssPLugins[i].name === "rucksack") { %>
+          , "rucksack-css": "0.8.x"
```

Comment:
@kylemac the prepended `,` doesn't work for json - any ideas on a better way to structure this?